arkouda/pdarraycreation.py

```diff
@@ -609,19 +601,22 @@ def randint(low : Union[int,float], high : Union[int,float], size : int, dtype=i
     return create_pdarray(repMsg)
 
 @typechecked
-def uniform(size : int, low : float=0.0, high : float=1.0, seed: Union[None, int]=None) -> pdarray:
+def uniform(size : int, low : float=0.0, high : float=1.0, 
+                                       seed: Union[None, int]=None) -> pdarray:
     """
-    Generate a pdarray with uniformly distributed random values 
+    Generate a pdarray with uniformly distributed random float values 
     in a specified range.
 
     Parameters
     ----------
     low : float
-        The low value (inclusive) of the range
+        The low value (inclusive) of the range, defaults to 0.0
     high : float
-        The high value (inclusive) of the range
+        The high value (inclusive) of the range, defaults to 1.0
     size : int
         The length of the returned array
+    seed : int
+        Index for where to pull the first returned value
```

Comment:
This should say "Value used to initialize the random number generator".

Reply:
updated